modules/redis/src/main/scala/zio/redis/api/Connection.scala

```diff
@@ -101,6 +101,15 @@ trait Connection[G[+_]] extends RedisEnvironment[G] {
     command.run(name)
   }
 
+  /**
+   * https://redis.io/docs/latest/commands/ping/
+   */
```

Comment:
```suggestion
  /**
   * @param message
   *   expected response payload
   *  
   * @return
   *   PONG if no argument is provided, otherwise return a copy of the argument
   */
```